scripts/helpers.js

```diff
@@ -71,6 +71,28 @@ hexo.extend.helper.register('blog_date_subtract_week', function (date) {
   return moment(date).subtract({weeks: 1}).format('MMM D[,] YYYY');
 });
 
+/**
+ * #1000 -> github.com/aframevr/aframe/pull/1000
+ * abcde -> github.com/aframevr/aframe/commit/abcde
+ * aframevr/aframe-editor|#1000 -> github.com/aframevr/aframe-editor/pull/1000
+ */
+hexo.extend.helper.register('github_contribution', function (contribution, display) {
+  var project = 'aframevr/aframe';
```

Comment:
`var project = this.config.github.aframe.username + '/' + this.config.github.aframe.repo;`